libraries/vmmanager/include/koinos/vmmanager/context.hpp

```diff
@@ -0,0 +1,22 @@
+#pragma once
+
+#include <koinos/vmmanager/api_handler.hpp>
+
+#include <cstdint>
+
+namespace koinos::vmmanager {
+
+/**
+ * A class containing information which may change execution of a VM.
+ */
+class context
+{
+   public:
+      context( api_handler& handler, int64_t ticks )
+         : _api_handler(&handler), _meter_ticks(ticks) {}
+
+      api_handler*        _api_handler = nullptr;
```

Comment:
We should avoid raw pointers. We're using this as a reference so it should probably be a reference. You may need to delete the default constructor.